docs/analysis-workflows/data-types.md

```diff
@@ -0,0 +1,176 @@
+| ---------------- | ----------------------------------------- | ---------------- | ---------------- | ---------------------- |
+| \*.aln.cram      | CRAM file format and compressed with gzip | sequencing_reads | Sequencing Reads | DNA Seq Alignment      |
+
+### Aligned Reads Index
+
+Secondary files that are external index files for CRAM format files. CRAI files follow a naming convention to correspond to their matching CRAM file of the orginal filename suffixed with `.cram.crai`. Index files are required for selective access to genomic data inside a CRAM via CRAM slicing.
+
+#### File Types
+
+| Filename Pattern | Description                                                 | Analysis Type    | Data Category    | Generating Workflow(s) |
+| ---------------- | ----------------------------------------------------------- | ---------------- | ---------------- | ---------------------- |
+| \*.aln.cram.crai | Cram Index file format. Requires a corresponding CRAM file. | sequencing_reads | Sequencing Reads | DNA Seq Alignment      |
+
+## Quality Control Metrics
+
+Quality control metrics are collected and recorded at several checkpoints in the ARGO Analytics pipeline to ensure that only high-quality data is released. All QC Metrics are associated to a corresponding analysis/file set that they annotate.
+
+### Read Group QC
+
+file containing read group (lane) level QC metrics including --,--?.
+
+#### File Types
+
+| Filename Pattern       | Description                                    | Analysis Type | Data Category           | Generating Workflow(s) |
+| ---------------------- | ---------------------------------------------- | ------------- | ----------------------- | ---------------------- |
+| \*.ubam_qc_metrics.tgz | Generated by Picard CollectQualityYieldMetrics | qc_metrics    | Quality Control Metrics | DNA Seq Alignment      |
+
+### Alignment QC
+
+Files containing quality control metrics for aligned CRAM files.
+
+#### File Types
+
+| Filename Pattern   | Description                             | Analysis Type | Data Category           | Generating Workflow(s)                                 |
+| ------------------ | --------------------------------------- | ------------- | ----------------------- | ------------------------------------------------------ |
+| \*.qc_metrics.tgz  | Generated by Samtools stats             | qc_metrics    | Quality Control Metrics | DNA Seq Alignment                                      |
+| \*.bas_metrics.tgz | Generated by Sanger bam_stats.pl script | qc_metrics    | Quality Control Metrics | Sanger WGS Variant Calling, Sanger WXS Variant Calling |
+
+### Duplicates Metrics
+
+Data file containing duplicates metrics for aligned CRAM files. Multiple reads that match at the same position in the genome are located and tagged duplicate reads in the CRAM file, where duplicate reads are defined as originating from a single fragment of DNA.
```

Comment:
if you change above introductions as above, could reword this to
`This is the data file containing duplicates metrics for aligned CRAM files.`

Reply:
no i there where is correct here